stock_dashboard_api/views/stocks_data_view.py

```diff
@@ -6,7 +6,7 @@
 from stock_dashboard_api.models.stock_data_models import StockData
 from stock_dashboard_api.utils.json_parser import middleware_body_parse_json
 mod = Blueprint('stocks_data', __name__, url_prefix='/stocks_data')
-
+test_mod = Blueprint('test_stocks_data', __name__, url_prefix='/stocks_data')
```

Comment:
Why it's called test_mod?

Reply:
it's called test_mod to show that this  Blueprint used only in tests